### PR TITLE
docs(legacy): deploy legacy docs sites

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: scripts/build-docs.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Once it's finished, go to the Netlify team we have, and update the manual deployments, if needed.
+
+If you need to add a new major legacy version, just add it to 'majors' configuration object,
+create a new deployment in Netlify, and finally add a CNAME in Cloudflare.
+`);
+
+const majors = {
+  v1: {
+    buildCommaid: 'yarn build-storybook',
```
Comment (sandgraham) [inline scripts/build-docs.js]: nit, `buildCommaid` -> `buildCommand`?

File: scripts/build-docs.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+function getLatestMajors(releases) {
+  return {
+    v1: releases.find(versionFilter('v1')),
+    v2: releases.find(versionFilter('v2')),
+    v3: releases.find(versionFilter('v3')),
+    v4: releases.find(versionFilter('v4')),
+    v5: releases.find(versionFilter('v5')),
+  };
```
Comment (sandgraham) [inline scripts/build-docs.js]: could this use the config object? or should this stay separate?

File: scripts/build-docs.js (hunk 1 above)
Comment (gergelyke) [reply scripts/build-docs.js]: ah, yeah, thanks!

File: scripts/build-docs.js (hunk 2 above)
Comment (gergelyke) [reply scripts/build-docs.js]: that's a good catch, let me update

**Pull request merged**